src/components/Table.vue

```diff
@@ -75,30 +75,35 @@
 </template>
 
 <script>
-import moment from 'moment'
+import HumanizeDate from '@videsk/humanize-date'
 
 export default {
   name: 'VueTable',
-    async mounted() {
+  async mounted() {
     // Users
     await this.getUsers();
 
   },
-    data() {
+  data() {
     return {
       users: [],
     };
   },
   filters: {
-    formatDate: (value) => moment(value).format('MMMM d, YYYY'),
-    formatDateRelative: (value) => moment(value).fromNow()
-    
+    formatDate: value => new HumanizeDate().toLocale(value),
+    formatDateRelative: (value) => new HumanizeDate().dates(value, new Date()).ago('months'),
   },
   methods: {
-     async getUsers() {
+    async getUsers() {
       const userResponse = await fetch("/api/users");
       const resp = await userResponse.json();
-      this.users = resp.users;
+      this.users = this.sortSessions(resp.users);
+    },
+    sortSessions(users) {
+      return users.map((user) => {
+        user.sessions = user.sessions.sort((a,b) => new Date(b) - new Date(a));
+        return user;
+      });
```

Comment:
Here you can sort and get the latest session! In the major of cases, this data will be sorted by backend but you need to ensure that! There are other techniques most sophisticated can you check googling :smile: